.gitlab/install_script_testing.yml

```diff
@@ -0,0 +1,19 @@
+---
+test_install_script:
+  stage: install_script_testing
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["arch:amd64"]
+  script:
+    - source /root/.bashrc
+    - set +x
+    - export GITLAB_TOKEN=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.gitlab_pipelines_scheduler_token --with-decryption --query "Parameter.Value" --out text)
+    - export APT_URL=$DEB_TESTING_S3_BUCKET
+    - export YUM_URL=$RPM_TESTING_S3_BUCKET
+    - export APT_REPO_VERSION_AGENT6="pipeline-$CI_PIPELINE_ID-a6 6"
+    - export YUM_VERSION_PATH_AGENT6="testing/pipeline-$CI_PIPELINE_ID-a6/6"
+    - export APT_REPO_VERSION_AGENT7="pipeline-$CI_PIPELINE_ID-a7 7"
+    - export YUM_VERSION_PATH_AGENT7="testing/pipeline-$CI_PIPELINE_ID-a7/7"
+    - inv pipeline.trigger-child-pipeline --project-name "DataDog/agent-linux-install-script" --git-ref "feat/install-script-test-on-release" --variables "APT_URL,YUM_URL,APT_REPO_VERSION_AGENT6,APT_REPO_VERSION_AGENT7,YUM_VERSION_PATH_AGENT6,YUM_VERSION_PATH_AGENT7"
```

Comment:
Shouldn't we make this test mandatory with the `--follow` option (even if it increases the pipeline duration, on deploy it's maybe acceptable)

Reply:
You're right I think it's better if we use `--follow` so we can see in the pipeline if it failed

Reply:
Actually default value for `follow` is true in the invoke task. For other jobs we usually set `--no-follow` to child pipeline we do not want to follow. So I can probably let it as is and the child pipeline is followed

Reply:
you're right, thanks!